specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/edgeorder.json

```diff
@@ -0,0 +1,3133 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConfigurationsRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of configurations for the given product family, product line and product under subscription.",
+            "schema": {
+              "$ref": "#/definitions/Configurations"
+            }
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListConfigurations": {
+            "$ref": "./examples/ListConfigurations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/productFamiliesMetadata": {
+      "get": {
+        "tags": [
+          "ProductFamilyMetadata"
+        ],
+        "description": "This method provides the list of product families metadata for the given subscription.",
+        "operationId": "ListProductFamiliesMetadata",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "description": "$skipToken is supported on list of product families metadata, which provides the next page in the list of product families metadata.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of available product families under subscription.",
+            "schema": {
+              "$ref": "#/definitions/ProductFamiliesMetadata"
+            }
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListProductFamiliesMetadata": {
+            "$ref": "./examples/ListProductFamiliesMetadata.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrder/productFamiliesMetadata/{productFamilyName}": {
```

Comment:
The RPC requires a collection GET at RG level (except for nested type). Please fix the lint error.
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource